app/api_clients/curation_api_client.py

```diff
@@ -0,0 +1,38 @@
+import uuid
+import requests
+import json
+from datetime import date
+
+from app.models.corpus_item_model import CorpusItemModel
+from app.models.ranked_corpus_items_instance import RankedCorpusItemsInstance
+
+
+class CurationAPIClient(object):
+    @classmethod
+    async def get_ranked_corpus_items(cls, corpus_id: str, start_date: str=None, user_id=None):
+        ranked_corpus_items_id = "NEW_TAB_EN_US"
+        if not start_date:
+            start_date = date.today().strftime("%Y-%m-%d")
+
+        query = f"""
+        query Query {{
```

Comment:
Can we name the query, this is the name that will show up in Apollo Studio when we look at metrics segmented by clients.

(metaflow example https://github.com/Pocket/dl-metaflow-jobs/pull/80)

Reply:
Done :)